packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -694,6 +696,53 @@ class AndroidGradleBuilder implements AndroidBuilder {
       color: TerminalColor.green,
     );
   }
+
+  @override
+  Future<List<String>> getBuildOptions({required FlutterProject project}) async {
+    final Status status = _logger.startProgress(
+      "Running Gradle task 'getBuildOptions'...",
+    );
+    final List<String> command = <String>[
+      _gradleUtils.getExecutable(project),
+      '-q',
```

Comment:
Consider adding a comment `'-q', // suppresses gradle output.` for readability and because the rest of your code assumes no gradle logging.